composer.json

```diff
@@ -119,8 +120,9 @@
             "cache:clear": "symfony-cmd",
             "assets:install %PUBLIC_DIR%": "symfony-cmd"
         },
-        "test": "phpunit --bootstrap ./tests/php/bootstrap.php",
-        "test:ci": "@test --log-junit ./build/phpunit.xml --coverage-clover ./build/coverage-be.xml --no-interaction --do-not-cache-result",
+        "pest": "./vendor/bin/pest",
+        "pest:coverage": "./vendor/bin/pest --coverage",
```

Comment:
coverage needs to be managed in `test:ci` command
```suggestion
```

Reply:
added